run.py

```diff
@@ -319,7 +319,7 @@ def synthesis(
         tags=["音声合成"],
         summary="音声合成する（キャンセル可能）",
     )
-    def cancellable_synthesis(
+    async def cancellable_synthesis(
```

Comment:
```suggestion
    def cancellable_synthesis(
```
ここを非同期にすると通信切断の検知ができなくなるかもしれません。
（音声合成が終わるまで制御がfastapiに戻らないため）
なのでここは同期のままが良さそうです。

Reply:
なるほどです！！たしかに、音声合成をキャンセルできなくなりそう。
こんな感じのFIXMEコメントを追加したいかもです。
```suggestion
    def cancellable_synthesis( # FIXME: 同時実行するとエラーになるが、通信切断を検知したいのでasyncにできない
```

Reply:
確かにそうですね、直しました